### DISCUSSION TITLE
Youtube live

### DISCUSSION THREAD

**Discussion by HalowTV** [streamlink/streamlink#5256] [General]
### Description

Hi i try run Youtube live stream like so on ubuntu 18:04 ssh get this issus 
streamlink -p mpv "https://www.youtube.com/watch?v=hGrU_eiV8-4" best --player-external-http --player-external-http-port 1002

[cli][info] streamlink is running as root! Be careful!
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=hGrU_eiV8-4
error: Unable to open URL: https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1679269786/ei/OksXZKKFNtTWgAfLi5jgDA/ip/2a01%3A4f8%3A13b%3A247%3A0%3A0%3A0%3A2/id/hGrU_eiV8-4.1/source/yt_live_broadcast/requiressl/yes/tx/24470261/txs/24470261%2C24470262%2C24470263%2C24470264%2C24470265%2C24470266%2C24470267/hfr/1/force_finished/1/maudio/1/vprv/1/go/1/pacing/0/nvgoi/1/keepalive/yes/fexp/24007246/dover/11/itag/0/playlist_type/CLEAN/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Ctx%2Ctxs%2Chfr%2Cforce_finished%2Cmaudio%2Cvprv%2Cgo%2Citag%2Cplaylist_type/sig/AOq0QJ8wRQIgP7LPHJBNyNFtoJDcUsnPV7OS7bdy8iVUtStEH4vCp8MCIQDWXFp6FiXJfKLXpCgcnbJk79H2dPuCzJM8mVVBAxHvwQ%3D%3D/file/index.m3u8 (429 Client Error: Too Many Requests for url: https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1679269786/ei/OksXZKKFNtTWgAfLi5jgDA/ip/2a01%3A4f8%3A13b%3A247%3A0%3A0%3A0%3A2/id/hGrU_eiV8-4.1/source/yt_live_broadcast/requiressl/yes/tx/24470261/txs/24470261%2C24470262%2C24470263%2C24470264%2C24470265%2C24470266%2C24470267/hfr/1/force_finished/1/maudio/1/vprv/1/go/1/pacing/0/nvgoi/1/keepalive/yes/fexp/24007246/dover/11/itag/0/playlist_type/CLEAN/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Ctx%2Ctxs%2Chfr%2Cforce_finished%2Cmaudio%2Cvprv%2Cgo%2Citag%2Cplaylist_type/sig/AOq0QJ8wRQIgP7LPHJBNyNFtoJDcUsnPV7OS7bdy8iVUtStEH4vCp8MCIQDWXFp6FiXJfKLXpCgcnbJk79H2dPuCzJM8mVVBAxHvwQ%3D%3D/file/index.m3u8)

**Comment by bastimeyer** [comment]
https://github.com/streamlink/streamlink/issues/5197
